_layouts/analytics.html

```diff
@@ -20,7 +20,9 @@ <h3><a href="{{ site.baseurl }}/api/analytics/{{ page.category }}/{{ page.days}}
     {%- if page.category == "os-version" -%}
       <code>{{ item.os_version }}</code>
     {%- elsif page.category == "cask-install" -%}
-      <code>{{ item.cask }}</code>
+      <a href="{{ site.baseurl }}/cask/{{ item.cask }}">
```

Comment:
This needs to check if the cask actually exists (e.g. versioned casks do not) like is done with formulae below.